serial_asyncio/__init__.py

```diff
@@ -19,6 +19,7 @@
 import os
 
 import serial
+from functools import wraps, partial
```

Comment:
Where is functools.wraps used in this change?

Reply:
Oops... only partial is used